streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -139,7 +139,7 @@ public class StreamsMetricsImplTest {
     private final MetricName metricName2 =
         new MetricName(METRIC_NAME1, CLIENT_LEVEL_GROUP, DESCRIPTION2, clientLevelTags);
     private final MockTime time = new MockTime(0);
-    private final StreamsMetricsImpl streamsMetrics = new StreamsMetricsImpl(metrics, CLIENT_ID, VERSION, time);
```

Comment:
Where does `VERSION` come from? Can we delete the variable completely?

Reply:
Yes! I have deleted it. There are `NODE_ID2` and `TOPIC_ID2` unused in the same file too.

Reply:
Feel free to delete them in this PR as side cleanup. (Small enough to just piggy-back it)